core/src/main/java/org/apache/iceberg/view/BaseView.java

```diff
@@ -128,4 +128,23 @@ public SQLViewRepresentation sqlFor(String dialect) {
 
     return closest;
   }
+
+  @Override
+  @SuppressWarnings("checkstyle:HiddenField")
+  public ReplaceViewVersion updateColumnDoc(String name, String doc) {
```

Comment:
Alternatively we could implement `UpdateSchema` for Views and the only implementation is updateColumndoc, and everything else throws Unsupported. I was -1 that since even returning UpdateSchema to callers will make it seem like some sort of schema evolution on views is possible, but it's not.

Reply:
Ah the awkward thing here is method chaining....

view.updateColumnDoc("col1", "docs").updateColumnDoc("col2", "docs").commit() will only update column2.

Maybe this should be updated to allow multiple column/docs

Reply:
Ah the awkward thing is method chaining...

```
view.updateColumDoc("col1", "doc1").updateColumnDoc("col2, "doc2").commit()
```

This will only update col2 since we're not doing any change tracking or re-using any previous replaces.

Maybe this API instead can accept multiple col/docs